### DISCUSSION TITLE
n-gon treasury

### DISCUSSION THREAD

**Discussion by tbxyd** [landgreen/n-gon#352] [General]
I've compiled a ton of n-gon related links into a readme file, and I'm going to create a website that holds them. if anyone has any additional links they want added, just lmk in this discussion 
https://github.com/c-rxxp-y/n-gon-treasury

**Comment by thesatelliteever** [comment]
https://github.com/thesatelliteever/ngonmodtestinggrounds heres mine i guess (im only getting started out with n-gon modding but hey its not bad for a first time)

**Comment by tbxyd** [comment]
Ill add it when i get a chance today, cool mod so far. If you check out mine, I've also added a sniper with some tech for it. One suggestion i do have, is that if you continue making mods i would put them into actual game code and not just a bookmarklet


Sent from my Apple Watch
